backend/config/config.exs

```diff
@@ -8,7 +8,8 @@
 use Mix.Config
 
 config :til,
-  ecto_repos: [Til.Repo]
+  ecto_repos: [Til.Repo],
+  frontend_host: System.get_env("FRONTEND_HOST")
```

Comment:
I need to get familiar with the project first, what is the purpose of `FRONTEND_HOST`?

Reply:
It is just our frontend app url, i use it to redirect after authorization - i can hardcode it if you think that it will be better.

Reply:
nope, it's fine via env